polyfill/test/exports.mjs

```diff
@@ -19,7 +19,7 @@ import * as Temporal from 'tc39-temporal';
 describe('Exports', () => {
   const named = Object.keys(Temporal);
   it(`should be 9 things`, () => {
```

Comment:
Change the test message to match the code as well. I'd also recommend adding a test below for "should contain 'parse'".

Reply:
:100: